### PR TITLE
⚡ 459 - UI remove tailwind classes

### LINKED ISSUE #459: 💄UI - Remove Tailwind Classes from MDX TableLayouts
Currently, in `software-audit.mdx` there are Tailwind classes applied to the `<TableLayout>` component. This is not ideal, and ideally we would have a set of preprogrammed Tailwind classes for non-tech people to select from, not individually modifiable Tailwind classes.

### PR DESCRIPTION
**Pull request by johnxu16**
Fixes #459 

Affected routes:

- /consulting/software-audit
- /terms-and-conditions

image
**Figure: Table on [https://ssw.com.au/consulting/software-audit](https://ssw.com.au/consulting/software-audit)**

image
**Figure: Table on local with presets**

image
**Figure: Set presets on Tina Admin**

image
**Figure: Edit preset on Tina Admin**

### CODE REVIEW

File: a [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+descendant-li:ml-6
+descendant-table:border-1
+descendant-table:border-solid
+descendant-table:p-2
+descendant-th:border-1
+descendant-th:border-solid
+descendant-th:p-2
+descendant-td:border-1
+descendant-td:border-solid
+descendant-td:p-2
```
Comment (Harry-Ross) [inline a]: What is this file for?

File: components/blocks/tableLayout.tsx [hunk 2]
```diff
@@ -1,34 +1,48 @@
...
+        (a, b) =>
+          presetNames.indexOf(a.node.name) - presetNames.indexOf(b.node.name)
+      );
+      const presetClassNames = orderedPresetNodes.map((p) => p.node.className);
+      setTableClasses(presetClassNames);
+    }
+
+    init();
+
+    const html = md.render(data.mdxTable ?? "");
```
Comment (Harry-Ross) [inline components/blocks/tableLayout.tsx]: Do we necessarily need to do a client-side GraphQL query here?

**Comment by johnxu16** [comment]
As per conversation with Harry, designers are not supposed see or configure tailwindcss directly.

- [x] Remove presets
- [x] Remove className on `TableLayout`

**Review by Harry-Ross — APPROVED** [review]
LGTM

**Pull request merged**